### PR TITLE
Slug for multilanguages

### PR DESCRIPTION
**Pull request by arthurtalkgoal**
Apply new keystone-utils slug method. 

The cld usually failed to find the locale, I allow ***autokey.defaultLocale*** to set up default locale.

### CODE REVIEW

File: lib/schemaPlugins/autokey.js [hunk 1]
```diff
@@ -94,9 +94,7 @@ module.exports = function autokey() {
 		if ((!modified || autokey.fixed) && this.get(autokey.path)) {
 			return next();
 		}
-
-		var newKey = utils.slug(values.join(' ')) || this.id;
-
+		var newKey = utils.slug(values.join(' '), null, {locale: autokey.defaultLocale}) || this.id;
```
Comment (JedWatson) [inline lib/schemaPlugins/autokey.js]: what do you think of just making this option `autokey.locale`?

**Comment by arthurtalkgoal** [comment]
Hi @JedWatson , that's good for me.

notice that cld2 actually just use it as "hints" not absolute locale setting.  Is it still ok to change the name to "locale"

**Comment by dcousens** [comment]
Does this resolve #1366?

**Comment by arthurtalkgoal** [comment]
Hi @JedWatson , I have renamed the field to ***locale***

Hi @dcousens , the #1366 is solving the same issue. Since the keystone-utils function definition has changed, so this keystone has changed accordingly.

**Comment by JedWatson** [comment]
Thanks @arthurtalkgoal, merged & will be released in a new version shortly.

**Pull request merged**